web/index.html

```diff
@@ -11,59 +10,76 @@
     <style>
       :root {
         color-scheme: light dark;
+
```

Comment:
Seems like your text editor change the code style.

We need to avoid it. Every PR should focus on single task.

Try to change text editor config and revert changes.